### PR TITLE
Fix: Cameras don't obstruct buttons anymore.

### PR DESCRIPTION
**Pull request by Scroody**
### What does this PR do?

This PR fixes the issue where the cameras were obstructing buttons.

### Closes Issue(s)

#17703

### CODE REVIEW

**Comment by trinhhuutam** [comment]
Should we use px instead of pt? That will fix the issue without * 0.75

**Comment by KDSBrowne** [comment]
If we use `px` here it will crash iphones

**Comment by trinhhuutam** [comment]
> If we use `px` here it will crash iphones. Maybe switch to `pt` only when iphones are detected.

Yes! Has tried with px and it causes crash on iPhone again. I'm using z index for (+) button to work-around this issue .

**Comment by antonbsa** [comment]
I just tested, no iPhone crashes were found! However, the whiteboard cursor is slightly offset when the cameras are positioned above or on the left of the presentation:


https://user-images.githubusercontent.com/56703993/234346862-be0acfdd-5654-4b26-9719-eda0213728c0.mp4

**Comment by trinhhuutam** [comment]
> dex for (+) button to work-around this issue when waiting official fix from you.

Btw, can you help to explain why use px here cause crash on Iphone only? I have to tried to find out but found nothing.
Does it cause crash on other elements with width height using px too?

File: bigbluebutton-html5/imports/ui/components/webcam/component.jsx [hunk 1]
```diff
@@ -154,7 +154,13 @@ const WebcamComponent = ({
   if (isRTL) {
     draggableOffset.left = draggableOffset.left * -1;
   }
+  const isIphone = !!(navigator.userAgent.match(/iPhone/i));
 
+  const mobileWidth = `${isDragging ? cameraSize.width : cameraDock.width}pt`;
+  const mobileHeight = `${isDragging ? cameraSize.height : cameraDock.height}pt`;
+  const isDesktopWidth = isDragging ? cameraSize.width : cameraDock.width;
+  const isDesktopHeight = isDragging ? cameraSize.height : cameraDock.height;
+  const camOpcity = isDragging ? 0.5 : undefined;
```
Comment (SamuelWei) [inline bigbluebutton-html5/imports/ui/components/webcam/component.jsx:163]:
```suggestion
  const camOpacity = isDragging ? 0.5 : undefined;
```

File: bigbluebutton-html5/imports/ui/components/webcam/component.jsx [hunk 2]
```diff
@@ -236,9 +242,9 @@ const WebcamComponent = ({
               role="region"
               draggable={cameraDock.isDraggable && !isFullscreen ? 'true' : undefined}
               style={{
-                width: `${isDragging ? (cameraSize.width * 0.75) : (cameraDock.width * 0.75)}pt`,
-                height: `${isDragging ? (cameraSize.height * 0.75) : (cameraDock.height * 0.75)}pt`,
-                opacity: isDragging ? 0.5 : undefined,
+                width: isIphone ? mobileWidth : isDesktopWidth,
+                height: isIphone ? mobileHeight : isDesktopHeight,
+                opacity: camOpcity,
```
Comment (SamuelWei) [inline bigbluebutton-html5/imports/ui/components/webcam/component.jsx:247]:
```suggestion
                opacity: camOpcity,
```
Fix typo, see other comment

**Review by SamuelWei — CHANGES_REQUESTED** [review]
Fix variable spelling typo

**Review by antonbsa — APPROVED** [review]
Re-tested and working as expected! No error logs or crashes, both on desktop and mobile

**Comment by jorgelondo** [comment]
Hi, someone can help me with the location of that file  "component.jsx" on the server to be able to make that change while the latest release comes out, because it's really urgent for me to make that change. thank you

**Pull request merged**